### PR TITLE
Fixes double eswords not having armor pen

### PR DESCRIPTION
**Pull request by FluffySurvivor**
as it should have happened in #912

an oversight on my part

:cl: FluffySurvivor 
bugfix: Double eswords are now properly granted armor penetration. 
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -55,6 +57,7 @@
 		return
 	wielded = 1
 	force = force_wielded
+	armour_penetration = armourpenetration_wielded
```
Comment (X-TheDark) [inline code/game/objects/items/weapons/twohanded.dm:61]: Remember to set 

```
armour_penetration = armourpenetration_unwielded
```

In the proc

```
/obj/item/weapon/twohanded/proc/unwield(mob/living/carbon/user)
```

Comment (FluffySurvivor) [inline code/game/objects/items/weapons/twohanded.dm:61]: it's a 0.3 damage difference, but sure

**Comment by oisin100** [comment]
EDIT: Nevermind, Dark got there before i could.

**Comment by FluffySurvivor** [comment]
fixed

**Comment by oisin100** [comment]
Possibly add a bugfix changelog.

**Comment by FluffySurvivor** [comment]
fixed x2

**Comment by oisin100** [comment]
Looking good, In future(Im gonna be saying this a lot more from now on...) please try and give commits valid names, it makes life easier for everyone.

**Pull request merged**